databuilder/contracts/universal.py

```diff
@@ -0,0 +1,31 @@
+from . import types
+from .base import Column, PatientIDColumn, TableContract
+from .constraints import FirstOfMonthConstraint, NotNullConstraint
+
+
+class Patients(TableContract):
+    _name = "patients"
+
+    patient_id = PatientIDColumn
+    date_of_birth = Column(
+        type=types.Date(),
+        description=(
+            "Patient's year and month of birth, provided in format YYYY-MM-01. "
+            "The day will always be the first of the month."
+        ),
+        constraints=[FirstOfMonthConstraint(), NotNullConstraint()],
+    )
+    sex = Column(
+        type=types.Choice("female", "male", "intersex", "unknown"),
+        description="Patient's sex as defined by the options: male, female, intersex, unknown.",
+        notes_for_users='Specify how this has been determined, e.g. "sex at birth", or "current sex".',
```

Comment:
In the doc you have three columns, which are mapped to the fields on Column like so:

| document | `Column` |
| ----- | ----- |
| Description for end users | `description` |
| Notes for implementers | `notes_for_implementers` |
| Notes implementers should consider supplying for end users | `notes_for_users`

For the Universal/Patients contract there are no columns with information in the `Notes for implementers` column, is that a mistake in the document?

Reply:
@ghickman I think that the structure of what you're provided is correct, but that the name "notes for users" is open to misinterpretation. It makes it sound as though the text in that field is intended for users, rather than being a description of the text that should be provided for users.

Reply:
(And Seb has accordingly misinterpreted it.)